### DISCUSSION TITLE
gpu_allocator is set to "pytorch" but Spacy is still saying "Using CPU"

### DISCUSSION THREAD

**Discussion by hp52** [explosion/spaCy#7077] [Help: Coding & Implementations] [answered]
Hi,
I 'm trying to do text classification using the GPU in Spacy 3.0. I set 
`gpu_allocator = "pytorch"`
in the config file. But when I run
`python -m spacy train`
it says
`ℹ Using CPU`
What do I have to change in order to make Spacy use the GPU?
Thanks for your help.

**Comment by svlandeg** [comment, marked as answer]
The [`train`](https://spacy.io/api/cli#train) command takes a setting `--gpu-id` or `-g` that defaults to -1 (a.k.a. CPU) - so you can set that when you want to train on GPU:
```
python -m spacy train ... -g 0
```

**Reply by hp52** [reply]
That worked, thanks.
